larq_zoo/model_factory.py

```diff
@@ -0,0 +1,49 @@
+from typing import Callable, Optional, Tuple, Union
+
+import tensorflow as tf
+from zookeeper import ComponentField, Field
+from zookeeper.tf import Dataset
+
+from larq_zoo import utils
+
+QuantizerType = Union[
+    tf.keras.layers.Layer, Callable[[tf.Tensor], tf.Tensor], str, None
+]
+ContraintType = Union[tf.keras.constraints.Constraint, str, None]
```

Comment:
```suggestion
ConstraintType = Union[tf.keras.constraints.Constraint, str, None]
```

However, does it make sense to define this type at all? It's only used once...